vendor/bat-native-ads/src/bat/ads/internal/federated/covariate_logs_unittest.cc

```diff
@@ -0,0 +1,46 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/federated/covariate_logs.h"
+
+#include "base/time/time.h"
```

Comment:
maybe use `unittest_time_util.h` as we can the use `Now()` and have access to other helpers for future use?

Reply:
fixed